pokemongo_bot/cell_workers/complete_tutorial.py

```diff
@@ -1,214 +0,0 @@
-import random
```

Comment:
@DeXtroTip Place different task classes in different files and then import them into the parent class.